cadl-extension/src/emitter.ts

```diff
@@ -7,7 +13,32 @@ import { CodeModelBuilder } from "./code-model-builder.js";
 import { dirname } from "path";
 import { fileURLToPath } from "url";
 
-export async function $onEmit(program: Program) {
+export interface EmitterOptions {
+  "namespace": string;
+  "service-name": string;
+  "partial-update": boolean;
+}
+
+const EmitterOptionsSchema: JSONSchemaType<EmitterOptions> = {
+  type: "object",
+  additionalProperties: false,
+  properties: {
+    "namespace": { type: "string", nullable: true },
+    "service-name": { type: "string", nullable: true },
+    "partial-update": { type: "boolean", nullable: true },
+  },
```

Comment:
we had high probability to add  `outputFolder` as well (I am not sure about the exact name though).

Also, should we use camelCase?

Reply:
Sure, will add that and change it to camel case.

Reply:
currently seems to be `outputPath` https://github.com/microsoft/cadl/issues/482

Reply:
Changed to outputPath.